main.go

```diff
@@ -19,19 +19,19 @@ func main() {
 	}
 	app := gin.Default()
 	config := &config.Config{}
-	if config.Project.Version == SupportedConfigVersion {
-		app.POST("/generate", func(ctx *gin.Context) {
-			ctx.BindJSON(config)
-			filename, err := generator.CreateFiles(*config, config.Project.Name, os.Getenv("STORAGE"))
+	app.POST("/generate", func(ctx *gin.Context) {
+		ctx.BindJSON(config)
+		filename, err := generator.CreateFiles(*config, config.Project.Name, os.Getenv("STORAGE"))
+		if config.Project.Version == SupportedConfigVersion {
```

Comment:
Currently, the statement is located on line 25, but it should be on line 24. Because we don't want to generate the files when config version is not supported.

Reply:
Right, thanks I made that change!